docs/gaia/gaiacli.md

```diff
@@ -128,6 +128,19 @@ gaiacli keys show --multisig-threshold K name1 name2 name3 [...]
 For more information regarding how to generate, sign and broadcast transactions with a
 multi signature account see [Multisig Transactions](#multisig-transactions).
 
+### Tx Broadcasting
+
+When broadcasting transactions, `gaiacli` accepts a `--broadcast-mode` flag. This
```

Comment:
maybe we can add a `-b` short mode for this?

Reply:
Sure.

Reply:
Done.